templates/shrine/src/shrine-app.html

```diff
@@ -223,7 +223,7 @@
           <app-toolbar class="profileBar">
             <img class="profilePic" src="//app-layout-assets.appspot.com/assets/shrine/profile_pic.jpg"  width="30" height="30">
             <div class="profileName">Stella</div>
-            <paper-icon-button icon="settings"></paper-icon-button>
+            <paper-icon-button icon="settings" alt="show settings"></paper-icon-button>
```

Comment:
@blasten I´ve used the `alt` attribute in this case according with the element documentation.